src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -209,6 +212,8 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
       </div>
     </fieldset>
   </form>
-  <alm-work-item-link [workItem]="workItem" [loggedIn]="loggedIn"></alm-work-item-link>
-  <alm-work-item-comment [workItem]="workItem" [loggedIn]="loggedIn"></alm-work-item-comment>
-</div>
+  <div *ngIf="!addNewWI">
```

Comment:
As per the wireframe, all the fields remain disabled until the title has been entered and saved. On the UI, the comments and link are not shown until the title is created.
Implementation wise, 
1. Create a work item with type + title
2. On save, we get the work item ID
3. We reload the details page with the newly created work item (this is the same as edit now)